wp-admin/js/widgets/media-widgets.js

```diff
@@ -200,7 +204,7 @@ wp.mediaWidgets = ( function( $ ) {
 				 *
 				 * @return {void}
 				 */
-				createStates: function() {
+				createStates: function createStates() {
```

Comment:
@westonruter can you explain the advantages or why you prefer this pattern as opposed to 'anonymous' functions?